codes/quantum/oscillators/fock_state/knill-laflamme-milburn-loqc.yml

```diff
@@ -0,0 +1,71 @@
+
+  general_gates:
+    - 'Single-qubit unitaries can be performed using purely linear optical elements. This is because any single qubit unitary U can be decomposed into rotations about the Z and Y axis in the Bloch sphere i.e. \( U=e^{i \alpha} R_{z}(\beta) R_{y}(\gamma) R_{z}(\delta)\). Arbitary Z and Y rotations which can be achieved using phase-shifters and beam splitters.'
+    - 'Universal gate set achieved by adding the controlled-Z phase (CZ) gate to the set of single-qubit unitaries.'
+    - 'A probabilistic controlled-Z gate is implemented by making use of the non-linear sign shift \((NS_{-1})\) gate, and then measuring the circuit adaptively to approximate linearity. The non-linear sign shift gate is defined by the following action: \( \alpha|0\rangle+\beta|1\rangle+\gamma|2\rangle \rightarrow \alpha|0\rangle+\beta|1\rangle-\gamma|2\rangle \)'
+    - 'The non-linear sign-shift gate \((NS_{-1})\) can be performed with a success probability of 1/4 \cite{arXiv:quant-ph/0512104}'
+    - 'Two non-linear sign-shift gates can perform the C-Sign gate with success probability 1/16. However it is not known whether these '
+    - 'Knill has shown that the CSign gate has an upper bound of 3/4 when using two ancilla modes, each with a maximum of 1 photon per mode.\cite{doi:10.1103/PhysRevA.68.064303}'
+    - 'The probability of success of the CZ gate (referred to in \cite{doi:10.1038/35051009} as the C-Sign gate) is improved using teleportation, by reducing the problem of construction of 2-qubit gates to a state preparation problem. We know when the state preparation fails, and can perform a correction.'
+    - 'The success probability of the CZ gate is further improved by using Z-measurement error-correction during teleportation.'
+
+  decoders:
+    - 'In case of Z-measurement errors i.e. \(Z \otimes \mathbb{1}\) and \(\mathbb{1} \otimes Z \), we perform syndrome measurement by measuring \(X \otimes X\), and apply Z-operation on the first and/or second qubit conditioned on the outcomes.'
+    - 'Similarly, a syndrome measurement circuit can be constructed for unintentional Y-measurement errors \cite{arXiv:quant-ph/0512104}'
+
+  fault_tolerance:
+    - 'Encoded X,Y,Z rotations can be performed transversally.'
+    - 'Teleportation-based 2-qubit gates can also be performed fault-tolerantly, by concatenating the code to protect against Z-measurement errors. If the error per qubit at one level of encoding is be less than the error per qubit at the next level of encoding we can keep this concatenated code and reduce the effective error rate, thereby implementing the circuit fault-tolerantly.'
+    - 'To ensure that fault tolerant error-correction procedure for teleportation reduces the error probability, we require the error probability of CSign gate,f, to be < 0.5 \cite{arXiv:quant-ph/0512104}'
+
+  code_capacity_threshold:
```

Comment:
I don't think the error model in the paper fits the description of a code capacity noise model so I would just delete this part (it is repeated in the threshold part anyway).